model/src/main/java/uk/gov/hmcts/payment/api/model/Payment2Repository.java

```diff
@@ -46,4 +46,14 @@ List<Reference> findReferencesByPaymentProviderAndPaymentStatusNotInAndDateCreat
     int updatePaymentUpdatedDateTime(@Param("rollbackdate") LocalDateTime rollbackDate,
                                    @Param("ccdcasenumber") String ccdCaseNumber);
 
+    @Query(value = "SELECT p.serviceType, p.ccdCaseNumber, p.reference, f.code, p.dateCreated, p.amount, p.paymentStatus.name " +
+        "FROM Payment p " +
+        "JOIN p.paymentLink pfl " +
+        "JOIN pfl.fees f " +
+        "WHERE p.dateCreated BETWEEN :fromDate AND :toDate AND p.paymentChannel.name = :paymentChannel")
+    List<Object[]> findAllByDateCreatedBetweenAndPaymentChannel(
+        @Param("fromDate") Date fromDate,
+        @Param("toDate") Date toDate,
+        @Param("paymentChannel") String paymentChannel);
+
 }
```

Comment:
Ollie we can avoid using Objects, may be we can do it 

EmployeeDepartmentDTO 

String hql = "SELECT new com.example.EmployeeDepartmentDTO(e.name, d.name) " +
             "FROM DeptEmployee e JOIN e.department d";
Query<EmployeeDepartmentDTO> query = session.createQuery(hql, EmployeeDepartmentDTO.class);
List<EmployeeDepartmentDTO> results = query.getResultList();

public class EmployeeDepartmentDTO {
    private String employeeName;
    private String departmentName;

    public EmployeeDepartmentDTO(String employeeName, String departmentName) {
        this.employeeName = employeeName;
        this.departmentName = departmentName;
    }
    // getters and setters
}